src/main/java/com/launchacademy/reviews/models/Word.java

```diff
@@ -41,6 +41,7 @@
   @JsonIgnoreProperties("word")
   private List<Review> reviews;
 
+
```

Comment:
If you remove this space, this file won't even have to be a part of this PR! 🎉 Yay clean git history!